src/components/About.tsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import { useStaticQuery, graphql } from 'gatsby';
+import Viaduct from './../images/viaduct.png';
+import Quebec from './../images/quebec.png';
+import styled from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faFilePdf } from '@fortawesome/free-solid-svg-icons';
+import { faGithub, faLinkedin } from '@fortawesome/free-brands-svg-icons';
+import ScrollingHeadline from './ScrollingHeadline';
+import {
+  Container,
+  Padding,
+  BlobMask,
+  Blobs,
+  SectionHeaderAlt,
+} from './../styles';
+import { Parallax } from 'react-scroll-parallax';
+
+const AboutContainer = styled(Container)`
+  align-items: center;
+  justify-content: center;
+`;
+
+const ImageContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  z-index: -3;
+
+  @media ${({ theme }) => theme.device.tablet} {
+    margin-bottom: 4rem;
+  }
+`;
+
+const ImageMaskA = styled(BlobMask)`
+  mask-image: url(${Blobs.aboutA});
+`;
+
+const ImageMaskB = styled(BlobMask)`
+  mask-image: url(${Blobs.aboutB});
+`;
+
+const TextContainer = styled.div`
+  margin-top: -4rem;
+  @media ${({ theme }) => theme.device.tablet} {
+    position: absolute;
+    left: 6rem;
+    max-width: 33rem;
+  }
+  @media ${({ theme }) => theme.device.laptop} {
+    left: 9rem;
+    bottom: 4rem;
+  }
+  @media ${({ theme }) => theme.device.highres} {
+    left: 22rem;
+    bottom: 0rem;
+  }
+`;
+
+const LinkContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  margin-top: 2rem;
+`;
+
+interface AboutProps {
+  id: string;
+}
+
+const About = ({ id }: AboutProps) => {
```

Comment:
It's common practice to annotate your components with Void Function Component or VFC for short and pass the interface through it. You won't need to annotate your param destructure.

```suggestion
import type { VFC } from 'react';

export const About: VFC<AboutProps> = ({ id }) => {
```

side note: you can have a named export here and just remove your default export below